bigquery_etl/cli/check.py

```diff
@@ -277,3 +292,6 @@ def _run_check(
 
     if check_failed:
         sys.exit(1)
+
+
+# todo: add validate method -- there must always be #fail checks
```

Comment:
Another TODO: We should add this to the data checks docs.

Reply:
I'll open an issue for this as well. We want something like `./bqetl checks validate` here